arctic/chunkstore/date_chunker.py

```diff
@@ -5,14 +5,16 @@
 
 
 class DateChunker(Chunker):
-    def to_chunks(self, df, chunk_size):
+    TYPE = 'date'
+
+    def to_chunks(self, df, chunk_size='D', **kwargs):
```

Comment:
I think `Y` or `M` might be a better default.  I found you get a big performance wins for larger and fewer chunks.